gdx/src/com/badlogic/gdx/math/Intersector.java

```diff
@@ -92,24 +96,36 @@ public static boolean intersectSegmentPlane (Vector3 start, Vector3 end, Plane p
 		return true;
 	}
 
-	/** Determines on which side of the given line the point is. Returns -1 if the point is on the left side of the line, 0 if the
-	 * point is on the line and 1 if the point is on the right side of the line. Left and right are relative to the lines direction
-	 * which is linePoint1 to linePoint2. */
+	/**
```

Comment:
IMO it is wasteful to use a whole line for this. The libgdx formatter will remove this line break. Also it's unreasonable to try to manually maintain code styling with a codebase as large as libgdx's.

Reply:
These words don't make sense. Line breaks don't "get closed". No idea when ensuring completeness or a doc generator is.